test/parallel/test-net-write-arguments.js

```diff
@@ -31,4 +39,10 @@ assert.throws(() => socket.write(null),
     message: 'The "chunk" argument must be of type string or an instance of ' +
               `Buffer.${common.invalidArgTypeHelper(value)}`
   }));
+  socket.on('error', common.expectsError({
+    code: 'ERR_INVALID_ARG_TYPE',
+    type: TypeError,
```

Comment:
I recently landed a change that removed this magic property. Using it will from now on fail when rebased to current master. It is possible to use `name: 'TypeError'` instead.

Reply:
Yep, noticed. Been sorting out all my PR's. Missed a few. Should be sorted now in this PR.